hydro_deploy/core/src/gcp.rs

```diff
@@ -337,9 +337,14 @@ impl Host for GcpComputeEngineHost {
         let vm_name = format!(
             "hydro-vm-instance-{}{}",
             nanoid!(8, &TERRAFORM_ALPHABET),
+            // Name must match regex: (?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)
+            // So no uppercase, no colons, no underscores, and no more than 61 chars
             self.display_name
                 .clone()
-                .map_or(String::new(), |name| format!("-{}", name))
+                .map_or(String::new(), |name| format!(
```

Comment:
Err, I would prefer the long but absolute names that include all the `::`, just split by `-` instead. For larger deployments we might see locations with the same name but different modules.

Reply:
You mean just replace :: with -?
I tried doing that but the name still exceeded 61 chars